docs/proposal-01-multi-profile-repo.md

```diff
@@ -80,7 +80,7 @@ $ tree weaveworks-profile/foo
 The `profile.yaml` would contain a reference to the artifacts relative to profile directory, for example:
 ```yaml
 $ cat foo/profile.yaml
-apiVersion: profiles.fluxcd.io/v1alpha1
+apiVersion: packages.weave.works/v1alpha1
```

Comment:
don't we need to change all the CRDs here too?

Reply:
I was also surprised by this, but no. Group version is still weave.works.... I'm not even sure where profiles here is coming from. I think we just put that there? It could just as well be groupname/version. Regardless, this change here, does not affect code under api or examples.

Reply:
Group is `weave.works`. And the CRD is named `  name: profilecatalogsources.weave.works` for example. That didn't reference `profiles` at all to being with. Nor fluxcd.io for that matter. So I don't think we need a code change there, but please feel free to correct me, if I see this incorrectly.

Reply:
ahhh yes your correct!